core-bundle/contao/classes/Backend.php

```diff
@@ -919,20 +919,24 @@ public static function getDcaPickerWizard($extras, $table, $field, $inputName)
 	 */
 	public static function getTogglePasswordWizard($inputName)
 	{
-		return ' ' . Image::getHtml('visible.svg', '', 'title="' . $GLOBALS['TL_LANG']['MSC']['showPassword'] . '" id="pw_' . $inputName . '"') . '
+		return ' <a href="#" title="' . StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['showPassword']) . '" id="pw_' . $inputName . '">' . Image::getHtml('visible.svg') . '</a>
```

Comment:
```suggestion
		return ' <button title="' . StringUtil::specialchars($GLOBALS['TL_LANG']['MSC']['showPassword']) . '" id="pw_' . $inputName . '">' . Image::getHtml('visible.svg') . '</button>
```
I think a button element would be semantically more correct here. 
_(But I’m not sure if this may create issues with the styling)_

Reply:
> (But I’m not sure if this may create issues with the styling)

It does:
![image](https://user-images.githubusercontent.com/1247552/221054584-716d56e2-b42e-4065-99ab-82cb4625f0dd.png)

Reply:
Fixing the CSS could probably be done by just setting `all: unset;` on the `<button>`